BuildingThemes/Detour/PoliciesPanelDetour.cs

```diff
@@ -163,7 +163,12 @@ private static void AddThemesTab()
 
             };
 
-
+            // Add a button to show the Building Theme Manager
+            UIButton showThemeManager = GUI.UIUtils.CreateButton(container);
+            showThemeManager.width = container.width;
+            showThemeManager.text = "Building Theme Manager";
+
+            showThemeManager.eventClick += (c, p) => GUI.UIThemeManager.instance.Toggle();
```

Comment:
I don't think it's necessary. Also clicking the button again will close the window.